### ISSUE TITLE
How to use own dataset for fully unsupervised learning

### ISSUE THREAD

**Issue by xiaohhuiiiii** [xu-ji/IIC#115]
First of all, thank you very much for your contribution. I have some doubts when I read the paper. The training data set used in the Auxiliary overclustering section of Section 3.2 includes two parts, one known to contain only relevant classes and the other known to contain irrelevant or distractor classes. Isn't there no label in unsupervised learning? Why do I know this information? Maybe I didn't understand it thoroughly. I hope you can give me some suggestions, thank you very much!!!
